src/components/popover/PopOver.tsx

```diff
@@ -50,9 +50,9 @@ export const PopOver = ({
       }
       backgroundStyle={style.backdrop}
     >
-      <View style={style.heading}>
+      <View style={style.heading} testID="popover">
```

Comment:
This testID is not used as far as I can see?

Reply:
That's true, but I added it to later be able to add tests for popovers, as well :-)

Reply:
Aha 😄

Reply:
A second thought, if I'll add popover tests I rather need a test-id for the specific popover text. So I'll remove this and rather add proper test-id later 👍